### ISSUE TITLE
Rule request: Line comment formatting rule

### ISSUE THREAD

**Issue by joshfriend** [realm/SwiftLint#1310]
I'd like to have a lint rule which validates the format of line and linline comments similar to [Python's PEP8 inline comment rule](https://www.python.org/dev/peps/pep-0008/#inline-comments). 

### Inline comments
Correct:
```swift
statement()  // some comment
```

Incorrect:
```swift
statement()//some comment
statement() //some comment
statement() // some comment
```

### Single line comments
This rule would be similar to the `MARK` rule added in #768, but would apply to _all_ line comments.

Correct:
```swift
// some comment
statement()
```

Incorrect:
```swift
//some comment
statement()
//  some comment
statement()
```

### Example configuration
This would be my proposal for the default values based on the Python standard and my personal preferences:

```yaml
inline_comment:
  spaces_before: 2
  spaces_after: 1
line_comment:
  spaces_after: 1
```

I think `1` would also be an acceptable default for `spaces_before`.

**Comment by kimdv** [comment]
@joshfriend , I'm currently working on this, and hope I can get a version ready this weekend :)

**Comment by Uncommon** [comment]
Comments sometimes are indented, so I think there should be a way to treat `spaces_after` as a minimum.

**Comment by joshfriend** [comment]
Thats a good point, commented out sections of code would certainly cause multiple violations of this rule. I was mostly hoping to make the spacing before and after `//` consistent throughout code and using a `spaces_after` setting as a minimum would probably not interfere with that goal

**Comment by maximkrouk** [comment]
Maybe it's not a right place to ask(pls redirect me if there is a better place), but can I ban multiline comments using swiftlint? 🙂
